### ISSUE TITLE
Playing a Level With the Noise Satellite Crashes the Game

### ISSUE THREAD

**Issue by poppip11** [GithubSPerez/pt-new-level-editor#40]
pretty self explanatory

crash log:
___________________________________________
############################################################################################
ERROR in
action number 1
of  Step Event0
for object obj_noisesatellite:

Variable obj_noisesatellite.stuntouchbuffer(100620, -2147483648) not set before reading it.
at gml_Script_scr_enemy_stun
############################################################################################
gml_Script_scr_enemy_stun (line -1)
gml_Object_obj_noisesatellite_Step_0

**Comment by 06NinjaKid06** [comment]
yeah my friend made level with it in it back in beta 1 and it crashed and he couldn't find out how to change instance layer to delete it  so that level was scrapped
